src/lib/serviceRoutinesV2/getEntityAttributeValue.cpp

```diff
@@ -98,17 +109,24 @@ std::string getEntityAttributeValue
         answer = attribute.pcontextAttribute->compoundValueP->render(ciP, JSON, "");
         if (attribute.pcontextAttribute->compoundValueP->isObject())
         {
-            answer = "{" + answer + "}";
+          TIME_STAT_RENDER_START();
+          answer = "{" + answer + "}";
```

Comment:
You are so right. I didn't see the call to the render method above.
Good catch!